config.js

```diff
@@ -21,5 +33,23 @@ module.exports = {
     redisHost: process.env.REDIS_ONLINELIST_PORT_6379_TCP_ADDR || 'localhost',
     redisPort: +process.env.REDIS_ONLINELIST_PORT_6379_TCP_PORT || 6379,
     maxSize: +process.env.ONLINE_LIST_SIZE || 20
+  },
+
+  pushApi: {
+    redisHost: process.env.REDIS_PUSHAPI_PORT_6379_TCP_ADDR || 'localhost',
+    redisPort: +process.env.REDIS_PUSHAPI_PORT_6379_TCP_PORT || 6379,
+    tokensPrefix: [unversionedApi, 'push-tokens'].join(':'),
+    notificationsPrefix: [unversionedApi, 'push-notifications'].join(':'),
+    apn: {
+      // connection
+      cert: process.env.APN_CERT_FILEPATH ||
+        path.join(__dirname, 'tests/push-api/cert.pem'),
+      key: process.env.APN_KEY_FILEPATH ||
+        path.join(__dirname, 'tests/push-api/key.pem'),
+      // push messages
+      expiry: Math.floor(Date.now() / 1000) + 3600,
```

Comment:
It would be more logical that expiry be calculated when sending the push notification (in case a worker runs for 1 hour without clearing the queue, this will fail)... I think `3600` only should be stored as a config option.